configurations/s02_pipeline_configuration.py

```diff
@@ -0,0 +1,200 @@
+                    FlowResultConfiguration("wusc_covid19_adaptation_kakuma_demogs", "Gender", "gender"),
+
+                    FlowResultConfiguration("wusc_keep_ii_s03_kakuma_demogs", "Household_Language",
+                                            "household_language"),
+                    FlowResultConfiguration("wusc_keep_ii_s03_kakuma_demogs", "Age", "age"),
+                    FlowResultConfiguration("wusc_keep_ii_s03_kakuma_demogs", "Location", "location"),
+                    FlowResultConfiguration("wusc_keep_ii_s03_kakuma_demogs", "Nationality", "nationality"),
+                    FlowResultConfiguration("wusc_keep_ii_s03_kakuma_demogs", "Gender", "gender")
+                ]
+            )
+        ),
+        RapidProSource(
+            rapid_pro=RapidProClientConfiguration(
+                domain="textit.com",
+                token_file_url="gs://avf-credentials/wusc-leap-kalobeyei-textit-token.txt"
+            ),
+            sync_config=RapidProToEngagementDBConfiguration(
+                flow_result_configurations=[
+                    FlowResultConfiguration("wusc_leap_s01_kalobeyei_demogs", "Household Language",
+                                            "household_language"),
+                    FlowResultConfiguration("wusc_leap_s01_kalobeyei_demogs", "Age", "age"),
+                    FlowResultConfiguration("wusc_leap_s01_kalobeyei_demogs", "Location", "location"),
+                    FlowResultConfiguration("wusc_leap_s01_kalobeyei_demogs", "Nationality", "nationality"),
+                    FlowResultConfiguration("wusc_leap_s01_kalobeyei_demogs", "Gender", "gender"),
+
+                    FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "Household_Language",
+                                            "household_language"),
+                    FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "Age", "age"),
+                    FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "Location", "location"),
+                    FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "Nationality", "nationality"),
+                    FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "Gender", "gender"),
+                    FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "Disability", "disability"),
+
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e01", "leap_s02e01")
+                ]
+            )
+        )
+    ],
+    coda_sync=CodaConfiguration(
+        coda=CodaClientConfiguration(credentials_file_url="gs://avf-credentials/coda-production.json"),
+        sync_config=CodaSyncConfiguration(
+            dataset_configurations=[
+                CodaDatasetConfiguration(
+                    coda_dataset_id="WUSC-KEEP-II_kakuma_gender", #TODO rename this to WUSC_kakuma_kalobeyei_gender
+                    engagement_db_dataset="kakuma_gender",
```

Comment:
This is a different name to that given in the fetch (kakuma_gender vs. gender).

I'm wondering what the best division is here - would it make sense to have separate databases for Kakuma/Kalobeyei and for Dadaab, given how hard configuration in the past has worked to keep them separate? Do we ever analyse the two locations together?

Reply:
A further thought on this is we should probably maintain a separate database because;
- We never analyze the two locations together
- Although rare we might have cross-talk between the camps and this might corrupt the demog data .
Have updated engagement_db_dataset to gender/location etc and table name to WUSC_KAKUMA_KALOBEYEI
-